src/host/rcp_host.cpp

```diff
@@ -846,6 +875,25 @@ void RcpHost::HandleEpskcStateChanged(void)
     }
 }
 
+#if OTBR_ENABLE_BACKBONE_ROUTER
+void RcpHost::HandleBackboneRouterMulticastListenerChanged(void                                  *aContext,
+                                                           otBackboneRouterMulticastListenerEvent aEvent,
+                                                           const otIp6Address                    *aAddress)
+{
+    static_cast<RcpHost *>(aContext)->HandleBackboneRouterMulticastListenerChanged(aEvent, aAddress);
+}
+
+void RcpHost::HandleBackboneRouterMulticastListenerChanged(otBackboneRouterMulticastListenerEvent aEvent,
+                                                           const otIp6Address                    *aAddress)
+{
+    VerifyOrExit(mBackboneRouterMulticastListenerCallback != nullptr);
+    mBackboneRouterMulticastListenerCallback(aEvent, Ip6Address(*aAddress));
```

Comment:
SafeInvoke?

Reply:
SafeInvoke is currently defined for type `AsyncResultReceiver`. Other callback type cannot be used with SafeInvoke. So we can only use `VerifyOrExit` here.